recipe/meta.yaml

```diff
@@ -76,7 +76,7 @@ outputs:
     script: install-win-wrapper.bat  # [win]
     requirements:
       build:
-        - python                                 # [build_platform != target_platform]
+        - m2-base  # [win]
       host:
         - {{ pin_subpackage('libxgboost', exact=True) }}
         - python
```

Comment:
Missing `wheel` in `host`

`setuptools` is not needed in `run`.
`scikit-learn` is an optional dependency , see https://github.com/dmlc/xgboost/blob/eb69c6110ae991ffcf05b60940e25ff8eb134a28/python-package/setup.py#L326

Reply:
It would be nice to add `pip check` in `test`

Reply:
Missing `license_family: Apache`

Reply:
> ... `scikit-learn` is an optional dependency , see https://github.com/dmlc/xgboost/blob/eb69c6110ae991ffcf05b60940e25ff8eb134a28/python-package/setup.py#L326

It seems we've always had this as a `run` dependency, so, I'm thinking we may not want to change it to optional.

Anyone else have any thoughts on this?

Reply:
I tend to agree with not removing features.